stdlib/ballerina-http/src/main/ballerina/http/http_commons.bal

```diff
@@ -14,24 +14,46 @@
 // specific language governing permissions and limitations
 // under the License.
 
+documentation {
+    Options for HTTP chunking.
+
+    `AUTO`: If the payload is less than 8KB, content-length header is set in the outbound response/response,
+            otherwise chunking header is set in the outbound response/response
+    `ALWAYS`: Always set chunking header in the response
+    `NEVER`: Never set the chunking header even if the payload is larger than 8KB in the outbound response/response
+}
 public type Chunking "AUTO" | "ALWAYS" | "NEVER";
```

Comment:
Move the descriptions of the values to the doc comments of the constants defined below

Reply:
And remove it from the union right?